package.json

```diff
@@ -3,18 +3,19 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
-    "@clr/icons": "^0.11.12",
+    "@clr/icons": "^0.11.14",
     "@webcomponents/custom-elements": "^1.1.0",
-    "lodash": "^4.17.5",
-    "rc-table": "^6.1.7",
-    "react": "^16.3.1",
+    "lodash": "^4.17.10",
```

Comment:
You're absolutely right; I wanted to update all the react stuff, since 16.3 is pretty recent and I wanted bugfixes for that (I made sure to look at patch notes for everything), but lodash and icons I just didn't bother to ignore.